packages/portfolio-contract/src/resolver/resolver.exo.ts

```diff
@@ -137,6 +158,25 @@ export const prepareCCTPResolver = (zone: Zone, vowTools: VowTools) => {
         getPendingTransactions() {
           return [...cctpTransactionRegistry.keys()];
         },
+        publishCctpTransactionStatus() {
+          const publishStatus = (status): void => {
+            void E.when(E(marshaller).toCapData(status), capData =>
+              E(cctpStatusNode).setValue(JSON.stringify(capData)),
+            );
+          };
+
+          const cctpTransactions = Array.from(
```

Comment:
This is O(n) vs an O(1) message. The contract **must** [Limit work by message size](https://github.com/Agoric/agoric-sdk/wiki/Limit-work-by-message-size).

The status of each tx should have its own node / key. It should go away once it's resolved, ideally.

Also: the parent node should get updated each time there's a new one, just like...

 - https://github.com/Agoric/agoric-sdk/pull/11688